pkg/justification/processor_test.go

```diff
@@ -188,6 +103,95 @@ func TestCreateToken(t *testing.T) {
 	for _, tc := range tests {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
+			t.Parallel()
+			var clientOpt option.ClientOption
+			key := "projects/[PROJECT]/locations/[LOCATION]/keyRings/[KEY_RING]/cryptoKeys/[CRYPTO_KEY]"
+			version := key + "/cryptoKeyVersions/[VERSION]"
+			keyID := key + "/cryptoKeyVersions/[VERSION]-0"
+			mockKeyManagement := &testutil.MockKeyManagementServer{
```

Comment:
Great we can make it parallel!

Could you do a refactor to make use of https://github.com/abcxyz/pkg/blob/main/testutil/grpc.go?